### PR TITLE
Attempt to add new services for MAUI SoilMapp.

### LINKED ISSUE #10: Need a few new entry points
The Apsoil web service has a rather long history, covering a period of time in which the underlying Apsoil format has changed. As a consequence, some of the functions expect data in one of the older formats, rather than the current one. As far as I can determine the different file formats include an old JSON version, an XML version, and an updated XML version. The older formats need to continue to be supported, as existing apps may rely on those formats. However, adding a few additional entry points to better support the "new" XML format will simplify development of new apps, such as an updated version of SoilMapp. Specifically, functions are needed to allow uploading of user soils in the "new' format, and also to allow the generation of PNG charts based on that format.

### PR DESCRIPTION
**Pull request by zur003**
@hol353 - I think these small changes should enable the additional services I need for the MAUI version of SoilMapp, but I'm not in a position to actually test whether it works. Also I don't know much about setting up web services, so I'm not sure whether the Reference.cs and Service.wsdl stuff gets updated automatically, or whether I need to modify them as well.

Resolves #10

### CODE REVIEW

**Comment by hol353** [comment]
@par456 or @ric394 Can you get this merged and get dev.apsim.info updated? The problem will be trying to compile this as it uses .NET framework 4.6.1 I think. It also wants to compile bits of APSIM 7.10. I think instead it should just link to the assemblies which we can commit to the repo.

**Comment by hol353** [comment]
[Apsim710Binaries.zip](https://github.com/APSIMInitiative/APSoil/files/14829268/Apsim710Binaries.zip)

File: WebService/Service.asmx.cs [hunk 1]
```diff
@@ -910,15 +933,24 @@ public SoilBasicInfo[] AllAustralianSoils(SearchSoilsParams Params)
...
+        /// MAUI soil app uses this call.
+        /// </summary>
+        [WebMethod]
+        public byte[] SoilChartPNGFromNewXML(string XML)
+        {
+            Soil Soil = Soil.Create(XML);
 
             SoilGraphUI Graph = CreateSoilGraph(Soil, false);
 
             //StreamWriter Out = new StreamWriter("D:\\Websites\\FILES\\Transfer\\ApsoilWeb.txt", true);
```
Comment (ric394) [inline WebService/Service.asmx.cs:950]: Commented code needs removing

**Review by ric394 — CHANGES_REQUESTED** [review]
Just need comments to be removed. Everything else looks good.

**Review by ric394 — APPROVED** [review]

**Pull request merged**